src/Modal/ModalContents.js

```diff
@@ -249,13 +251,9 @@ export default class ModalContents extends Util.mixin(BindMixin) {
 
   render() {
     let props = this.props;
-    let containerClass = {};
-    if (props.open) {
-      containerClass[props.containerClass] = true;
-    }
 
     return (
-      <div className={classNames(containerClass)}>
+      <div>
```

Comment:
Sure, I say let's add the optional class just in case. However I don't think it will be super useful because the element doesn't animate, and it will always be in the DOM, regardless of whether the modal is open or closed.